packages/web-components/src/components/button-group/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import ExpressiveModal from './ExpressiveModal';
+
+# Button Group
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/button-group)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/button-group)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/button-group/button-group';
+import '@carbon/ibmdotcom-web-components/es/components/button-group/button-group-item';
+```
+
+or with [JSPM](https://jspm.org)
+
+```html
+<script type="module">
+  import 'https://jspm.dev/@carbon/ibmdotcom-web-components@canary/es/components/button-group/button-group.js';
+  import 'https://jspm.dev/@carbon/ibmdotcom-web-components@canary/es/components/button-group/button-group-item.js';
+</script>
+```
+
+### HTML
+
+```html
+<dds-button-group>
+  <dds-button-group-item>Lorem ipsum</dds-button-group-item>
+  <dds-button-group-item>Dolor sit amet</dds-button-group-item>
+</dds-button-group>
+```
```

Comment:
@RaphaelAmadeu looks good! Can you update the README to show an example of including attributes like `href`, how to add an icon, etc?

Reply:
@jeffchew I've updated it on my latest pr